### DISCUSSION TITLE
vscode terminal disappears

### DISCUSSION THREAD

**Discussion by erhan9398** [microsoft/vscode-cpptools#7787] [General]
in ubuntu machine, i am using vscode with no problem, but after certain or random amount of time terminal disappears suddenly and it becomes just black screen on terminal part. when i disable and re-enable by using Ctrl+j terminal stays black then i add new terminal so that it can be used. the first terminal still stay black, thus i close first one.

**Comment by sean-mcmanus** [comment]
Our C/C++ extension doesn't control that behavior. You may want to file an issue at https://github.com/microsoft/vscode/issues .

**Reply by erhan9398** [reply]
thanks
